proto/cosmos/crisis/v1/tx.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+package cosmos.crisis.v1;
```

Comment:
If this is called circuit (ref: https://github.com/cosmos/cosmos-sdk/pull/14229#issuecomment-1351750658), shouldn't be renamed and the file moved to `proto/cosmos/circuit`?